pages/guide.py

```diff
@@ -223,18 +223,12 @@ def get_template_data(self, **defaults):
       ('tag_review', 'str'),
       ('tag_review_status', 'int'),
       ('webview_risks', 'str'),
-      ('comments', 'str'),
+      ('feature_notes', 'str'),
       ('breaking_change', 'bool'),
       ('ongoing_constraints', 'str')]
 
   # Old field name, new field name
   RENAMED_FIELD_MAPPING: dict[str, str] = {
-      'owner': 'owner_emails',
```

Comment:
What about users who have a tab open on editing page at the time that we deploy this change?  Won't they be submitting a POST form that has the old names that still need to be mapped?  We could hold of on deleting these mappings and deploy that part a week later.

Reply:
Good idea - I've added back the old mapping, so this scenario should not cause any problems for users